websites/S/Spyke/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "726466786916171796",
+		"name": "iamhv"
+	},
+	"service": "Spyke",
+	"description": {
+		"en": "Spyke - Community Discussion"
+	},
+	"url": [
+		"spyke.social",
+		"www.spyke.social"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/v10pkZA.png",
```

Image seems to be invalid or deleted.
```suggestion
	"logo": "https://i.imgur.com/vosdvpf.jpg",
```